### DISCUSSION TITLE
Default Username: birdnet

### DISCUSSION THREAD

**Discussion by salado912** [Nachtzuster/BirdNET-Pi#188] [Q&A]
is there a way to change the Default Basic Authentication Username: birdnet?
maybe in a config file somewhere

**Comment by alexbelgium** [comment]
I haven't tried but I see that it is hardcoded here :
https://github.com/Nachtzuster/BirdNET-Pi/blob/714519b35bb5bd98a80aadcd016f06436bddb8cc/scripts/common.php#L71

You might try to change it in this file
